courses/free-courses-ne.md

```diff
@@ -43,3 +46,8 @@
 
 * [WordPress Complete Tutorial In Nepali](https://www.youtube.com/playlist?list=PL2OJkQtHPRie2xyBApANdVp_LUz4v7xIG) - Technology Channel
 
+
+#### PHP
+
+* [PHP Basic Tutorial In Nepali](https://youtu.be/Ec8bwfCNhkA?si=WgnbspeofHlg65VE) - Technology Channel
```

Comment:
no shortened links

Reply:
done. now have included a complete playlist for web dev, and excluded shortened links